documentation/source/quickstart.rst

```diff
@@ -41,6 +41,11 @@ To install the development version of cocotb:
     git clone https://github.com/cocotb/cocotb
     pip3 install -e ./cocotb
 
+.. note::
+
+    After installation, you should be able to execute ``cocotb-config``.
+    If it is not found, you need to append its location to the ``PATH`` environment variable.
+    This may happen when you use the ``--user`` option to ``pip``, in which case the location is documented `here <https://docs.python.org/3/library/site.html#site.USER_BASE>`_.
```

Comment:
```suggestion
    This may happen when you use the ``--user`` option to ``pip``, in which case the location is documented `here <python:site.USER_BASE>`_.
```
Or something, will generate an intersphinx link.